src/components/pages/contact/Contact.jsx

```diff
@@ -24,25 +21,31 @@ export const Contact = () => {
             }, (error) => {
                 console.log(error.text);
             });
+    }
 
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        setFormValues(FORM_INITIAL_VALUES);
+        sendEmail();
         alert('Message has been sent!');
-        setEmail('');
-        setMessage('');
-        setName('');
     };
 
+    const onChangeHandler = (e) => {
+        const inputName = e.target.name;
+        setFormValues({ ...formValues, [inputName]: e.target.value })
+    }
+
     return (
         <ContactBackground>
             <StyledContainer>
                 <StyledH2>If you have any <span className='orange__font'>question</span><br />send the <span className='orange__font'>message</span>!</StyledH2>
```

Comment:
znalazłam kolejne br  :) 
jest gdzieś zdefiniowana klasa 'orange_font'?

Reply:
co to 'br'?

Reply:
Ten br jest ok, przenosi tylko tekst linię niżej, a nie oddziela labele od tekstu. 
orange_font zdefiniowany w App.css